test/views/declaration/DraftConfirmationViewSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.declaration
+
+import java.time.{LocalDateTime, ZoneOffset}
+
+import helpers.views.declaration.CommonMessages
+import models.responses.FlashKeys
+import play.api.mvc.Flash
+import play.twirl.api.Html
+import views.declaration.spec.ViewSpec
+import views.html.declaration.draft_confirmation_page
+import views.tags.ViewTest
+
+@ViewTest
+class DraftConfirmationViewSpec extends ViewSpec with CommonMessages {
+
+  private val page = app.injector.instanceOf[draft_confirmation_page]
```

Comment:
What do you think about move this to unit test ?

Reply:
Perhaps, all the view specs appear to follow this pattern though